beacon-node-oapi.yaml

```
@@ -65,6 +65,8 @@ paths:
 
   /eth/v1/beacon/genesis:
     $ref: "./apis/beacon/genesis.yaml"
+  /eth/v1/beacon/states/{state_id}:
+    $ref: "./apis/beacon/states/state.yaml"
```

```suggestion
  /eth/v1/debug/states/{state_id}:
    $ref: "./apis/beacon/states/state.yaml"
```

per comments elsewhere, this URL change is unnecessary and remains a cheap way to provide superior backwards compatibility.

---

its definitely not going to be `/eth/v1/debug/beacon/states/{state_id}` - that route was used, and Im not sure why we'd want a brand new route here `/eth/v1/debug/states/{state_id}`, can you clarify what you mean exactly?

just to be explicit, the current route that offers this is `/eth/v2/debug/beacon/states/{state_id}`, and the suggestion is that becomes `/eth/v1/beacon/states/{state_id}`

---

the practical outcome would be that json support continues to live in the debug section of the spec while ssz becomes "enshrined" under the beacon section (but with the same url for backwards compatibility)